src/createServer.js

```diff
@@ -1,8 +1,44 @@
-'use strict';
+/* eslint-disable no-console */
+"use strict";
+
+const http = require("node:http");
+const fs = require("fs");
 
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    if (req.url.includes("../")) {
+      res.writeHead(400, { "Content-Type": "text/plain" });
+      res.end("Invalid path");
+
+      return;
+    }
+
+    if (!req.url.startsWith("/file/")) {
+      res.writeHead(200, { "Content-Type": "text/plain" });
+      res.end("Hint: Routes must start with /file/");
+
+      return;
+    }
+
+    if (req.url.includes("//")) {
+      res.writeHead(404, { "Content-Type": "text/plain" });
```

Comment:
in order to make less duplicates maybe write `res.setHeader('Content-Type', 'text/plain');` one time in the beginning of the function